src/Dfe.PlanTech.Web/TagHelpers/HeadingComponentTagHelper.cs

```diff
@@ -18,45 +17,26 @@ public HeaderComponentTagHelper(ILogger<HeaderComponentTagHelper> logger)
 
     public override void Process(TagHelperContext context, TagHelperOutput output)
     {
-        if (Model == null)
+        if (Model == null || Model.Tag == Domain.Content.Enums.HeaderTag.Unknown || !TryBuildElement(output))
```

Comment:
Any reason we don't add `using Dfe.PlanTech.Domain.Content.Enums`, to change this to `Model.Tag == HeaderTag.Unknown`?